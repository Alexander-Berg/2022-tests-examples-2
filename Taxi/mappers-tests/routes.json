[
  {
    "expected": [
      {
        "archived_at": "2021-07-06T11:18:03.209129+00:00",
        "is_cyclic": true,
        "name": "moscow.test",
        "path": [
          [
            30.0,
            60.0
          ],
          [
            30.1,
            60.1
          ]
        ],
        "route_id": 123
      }
    ],
    "input": {
      "archived_at": {
        "$date": "2021-07-06T11:18:03.209129Z"
      },
      "is_cyclic": true,
      "name": "moscow.test",
      "path": [
        [
          30.0,
          60.0
        ],
        [
          30.1,
          60.1
        ]
      ],
      "route_id": 123
    }
  }
]
