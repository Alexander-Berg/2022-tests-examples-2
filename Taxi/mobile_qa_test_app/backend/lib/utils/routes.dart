class Routes {
  static const build = '/build/*';
  static const categories = '/categories';
  static const ping = '/ping';
}
