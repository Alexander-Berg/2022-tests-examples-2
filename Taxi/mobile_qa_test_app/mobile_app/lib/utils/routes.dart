import 'package:flutter/material.dart';
import 'package:models/models.dart';

import '../address/ui/page.dart';
import '../app_loader/ui/page.dart';
import '../auth/ui/auth/page.dart';
import '../auth/ui/code/ui/page.dart';
import '../auth/ui/phone/ui/page.dart';
import '../auth/ui/sign_up/ui/page.dart';
import '../build/ui/page.dart';
import '../card/ui/page.dart';
import '../cart/ui/page.dart';
import '../payment/ui/page.dart';
import '../payment_result/ui/page.dart';
import '../profile/ui/page.dart';
import '../shop/ui/category/page.dart';
import '../shop/ui/main/ui/page.dart';
import '../shop/ui/product/page.dart';
import '../support/ui/answer/page.dart';
import '../support/ui/questions/page.dart';

class AppRoutes {
  static const appLoader = '/';
  static const auth = '/auth';
  static const build = '/build';
  static const signUp = '/signUp';
  static const code = '/code';
  static const phone = '/phone';
  static const supportQuestions = '/supportQuestions';
  static const supportAnswer = '/supportAnswer';
  static const shop = '/shop';
  static const card = '/card';
  static const address = '/address';
  static const product = '/product';
  static const cart = '/cart';
  static const paymentResult = '/paymentResult';
  static const payment = '/payment';
  static const profile = '/profile';
  static const category = '/category';
}

Route<dynamic>? onGenerateRoutes(RouteSettings settings) {
  final args = settings.arguments;
  final Widget page;
  switch (settings.name) {
    case AppRoutes.appLoader:
      page = const AppLoaderPage();
      break;
    case AppRoutes.build:
      page = const BuildPage();
      break;
    case AppRoutes.auth:
      page = const AuthPage();
      break;
    case AppRoutes.signUp:
      page = const SignUpPage();
      break;
    case AppRoutes.supportAnswer:
      page = const AnswerPage();
      break;
    case AppRoutes.code:
      page = const CodePage();
      break;
    case AppRoutes.phone:
      page = const PhonePage();
      break;
    case AppRoutes.supportQuestions:
      page = QuestionsPage();
      break;
    case AppRoutes.shop:
      page = const ShopMainPage();
      break;
    case AppRoutes.address:
      page = const AddressPage();
      break;
    case AppRoutes.product:
      page = args == null
          ? const ShopMainPage()
          // ignore: cast_nullable_to_non_nullable
          : ProductPage(args as Product);
      break;
    case AppRoutes.cart:
      page = const CartPage();
      break;
    case AppRoutes.category:
      page = args == null
          ? const ShopMainPage()
          : ShopCategoryPage(args as Category);
      break;
    case AppRoutes.card:
      page = const CardPage();
      break;
    case AppRoutes.paymentResult:
      page = const PaymentResultPage();
      break;
    case AppRoutes.payment:
      page = const PaymentPage();
      break;
    case AppRoutes.profile:
      page = const ProfilePage();
      break;
    default:
      page = Container();
  }

  return MaterialPageRoute<void>(
    builder: (_) => page,
  );
}
