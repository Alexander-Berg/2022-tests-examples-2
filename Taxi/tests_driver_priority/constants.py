# generated via tvmknife unittest service -s 200 -d 300 (priority->tags)
TICKET_200_300 = (
    '3:serv:CBAQ__________9_IgYIyAEQrAI:RGXe3QVsvyGEMcOnW4VEqV78XIcLC4XXS7FTy'
    '-iafuLdLR2FY8UmScF7VOCVTh9e3YQfCQ9VsnscVMmdbmiSKDoHZofqGA-rACcc0rdGv9D81'
    'fpualqaL_O6nPEuXtTb7qpxQVjpX1EknAam_FQEKeyEG6KN51rcf4iwgQSsacg'
)

# generated via tvmknife unittest service -s 300 -d 200
TICKET_300_200 = (
    '_3:serv:CBAQ__________9_IgYIrAIQyAE:kqftMeryz0oR__X2DpAGWHga3cFV_y5FmPjY'
    'RASdxB-RY1PgC0aw-YoPQ6ZHU30Ojg07a3EJzWSuJoNdYmpztSiNz4Dn50TXjrDjTsl9CIrU'
    'ugnYYbFFbgrLZR8BmSctwhbObJM37z-IrZgyhM45Lw4kv80MezQ4Xt6r3oazUw'
)

BR_ROOT = 'br_root'
MSK = 'moscow'
SPB = 'spb'
TULA = 'tula'

DEFAULT_ZONES_WHITELIST = {'__default__': True, MSK: True}
DEFAULT_ACCEPT_LANGUAGE = 'en-US, en;q=0.8,ru;q=0.6'
DEFAULT_USER_AGENT = 'Taximeter 9.20 (1234)'
DEFAULT_HEADERS = {
    'Accept-Language': DEFAULT_ACCEPT_LANGUAGE,
    'User-Agent': DEFAULT_USER_AGENT,
}

POLLING_URL = 'v1/polling/priority'
DIAGNOSTICS_URL = 'v1/priority/diagnostics'
SCREEN_URL = 'v1/priority/screen'
VALUE_URL = 'v1/priority/value'
VALUES_URL = 'v1/priority/values'
FLEET_URL = 'fleet/driver-priority/v1/priority/diagnostics'

MSK_COORDS = (37.6, 55.75)
