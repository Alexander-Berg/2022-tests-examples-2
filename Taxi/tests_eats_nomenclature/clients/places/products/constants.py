SKU_UUID = '11111111-1111-1111-1111-111111111111'
HANDLER = '/v1/places/products/info-by-sku-id'
