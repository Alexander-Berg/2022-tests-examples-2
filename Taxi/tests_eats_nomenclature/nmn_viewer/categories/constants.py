ASSORTMENT_ID = 1
BRAND_ID = 1
PLACE_ID = 1
S3_PREFIX = 'viewer/new/categories/'
