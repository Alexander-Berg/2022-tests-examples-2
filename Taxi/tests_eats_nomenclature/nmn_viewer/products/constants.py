ASSORTMENT_ID = 1
BRAND_ID = 1
CATEGORY_ID = 1
S3_PREFIX = 'viewer/new/products/'
