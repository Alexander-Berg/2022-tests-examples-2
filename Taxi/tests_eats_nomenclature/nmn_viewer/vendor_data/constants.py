PLACE_ID = 1
BRAND_ID = 1
S3_PREFIX = 'viewer/new/vendor_data'
S3_PATH_PATTERN = f'{S3_PREFIX}/{BRAND_ID}/[a-f0-9]+.json'
