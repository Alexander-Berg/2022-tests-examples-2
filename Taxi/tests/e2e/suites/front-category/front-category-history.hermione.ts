import crypto from 'crypto';
import createImageFile from 'tests/e2e/utils/create-image-file';
import {describe, expect, it} from 'tests/hermione.globals';

import {ROUTES} from '@/src/constants/routes';

const FRONT_CATEGORY_ID = 87;
const ROOT_FRONT_CATEGORY_ID = 5;
const FRONT_CATEGORY_ID_IN_IL_REGION = 65;

describe('История изменений фронт-категории', function () {
    async function assertListItemImageStretched(ctx: Hermione.TestContext, selector: string) {
        const initialSize = await ctx.browser.getWindowSize();
        const container = await ctx.browser.findByTestId('history-of-changes_list');

        await container.execute(
            (container, selector) => {
                if (container instanceof HTMLElement) {
                    container.style.removeProperty('height');

                    [...container.querySelectorAll('[data-testid^=list-item')]
                        .filter((it): it is HTMLElement => it instanceof HTMLElement)
                        .filter((it) => it.dataset.testid !== selector)
                        .forEach((it) => it.style.setProperty('display', 'none', 'important'));
                }
            },
            container,
            selector
        );

        await ctx.browser.setWindowSize(initialSize.width, 1500);
        await ctx.browser.assertImage(['history-of-changes_list', selector]);
        await ctx.browser.setWindowSize(initialSize.width, initialSize.height);
    }

    it('Запись о создании пустой фронт-категории', async function () {
        await this.browser.openPage(ROUTES.CLIENT.CREATE_FRONT_CATEGORY);

        await this.browser.typeInto('code', 'some_front_category_code', {clear: true});
        await this.browser.typeInto(['translations', 'ru', 'name'], 'My front-category');
        await this.browser.clickInto('submit-button', {waitNavigation: true, waitRender: true});

        await this.browser.clickInto('\\#rc-tabs-0-tab-history-of-changes-tab', {waitForClickable: true});

        await this.browser.waitForTestIdSelectorInDom(['history-of-changes_list', '^list-item']);
        await assertListItemImageStretched(this, 'list-item-2');
    });

    it('Запись о замене названия фронт-категории', async function () {
        await this.browser.openPage(ROUTES.CLIENT.FRONT_CATEGORY(FRONT_CATEGORY_ID));

        await this.browser.typeInto(['translations', 'ru', 'name'], 'Renamed front-category', {clear: true});
        await this.browser.clickInto('submit-button', {waitRender: true});

        await this.browser.clickInto('\\#rc-tabs-0-tab-history-of-changes-tab', {waitForClickable: true});

        await this.browser.waitForTestIdSelectorInDom(['history-of-changes_list', '^list-item']);
        await this.browser.assertImage(['history-of-changes_list', 'list-item-2']);
    });

    it('Запись об удалении описания на английском фронт-категории. Израиль', async function () {
        await this.browser.openPage(ROUTES.CLIENT.FRONT_CATEGORY(FRONT_CATEGORY_ID_IN_IL_REGION), {region: 'il'});

        await this.browser.typeInto(['translations', 'en', 'description'], '', {clear: true});
        await this.browser.clickInto('submit-button', {waitRender: true});

        await this.browser.clickInto('\\#rc-tabs-0-tab-history-of-changes-tab', {waitForClickable: true});

        await this.browser.waitForTestIdSelectorInDom(['history-of-changes_list', '^list-item']);
        await this.browser.assertImage(['history-of-changes_list', 'list-item-2']);
    });

    it('Запись о смене статуса фронт-категории', async function () {
        await this.browser.openPage(ROUTES.CLIENT.FRONT_CATEGORY(FRONT_CATEGORY_ID));

        await this.browser.clickInto('disabled_label');
        await this.browser.clickInto('submit-button', {waitRender: true});

        await this.browser.clickInto('\\#rc-tabs-0-tab-history-of-changes-tab', {waitForClickable: true});

        await this.browser.waitForTestIdSelectorInDom(['history-of-changes_list', '^list-item']);
        await this.browser.assertImage(['history-of-changes_list', 'list-item-2']);
    });

    it('Поиск по истории изменений фронт-категории', async function () {
        await this.browser.openPage(ROUTES.CLIENT.FRONT_CATEGORY(FRONT_CATEGORY_ID));

        await this.browser.clickInto('\\#rc-tabs-0-tab-history-of-changes-tab', {waitForClickable: true});
        await this.browser.waitForTestIdSelectorInDom(['history-of-changes_list', '^list-item']);
        await this.browser.typeInto('history-of-changes_search', 'истин');

        await this.browser.assertImage('query-state-spinner');
    });

    it('Запись о смене всех полей фронт-категорий', async function () {
        await this.browser.openPage(ROUTES.CLIENT.FRONT_CATEGORY(ROOT_FRONT_CATEGORY_ID));

        await this.browser.typeInto('sort-order', '2', {clear: true});
        await this.browser.clickInto('promo');
        await this.browser.typeInto('deeplink', 'new_deeplink', {clear: true});
        await this.browser.typeInto('legal_restrictions', 'New Legal Restrictions', {clear: true});
        await this.browser.clickInto('delete-image');
        await this.browser.uploadFileInto(
            ['category_image', 'file-input'],
            createImageFile('new_front_category_image.png')
        );
        await this.browser.clickInto('submit-button', {waitRender: true});

        await this.browser.clickInto('\\#rc-tabs-0-tab-history-of-changes-tab', {waitForClickable: true});

        await this.browser.waitForTestIdSelectorInDom(['history-of-changes_list', '^list-item']);
        await this.browser.assertImage(['history-of-changes_list', 'list-item-2']);
    });

    it('Скачивание изображения из истории фронт-категории', async function () {
        await this.browser.openPage(ROUTES.CLIENT.FRONT_CATEGORY(ROOT_FRONT_CATEGORY_ID));
        await this.browser.clickInto('\\#rc-tabs-0-tab-history-of-changes-tab', {
            waitForClickable: true,
            waitRender: true
        });

        await this.browser.clickInto(['download-image'], {waitRender: true});
        const file = await this.browser.getDownloadedFile('front_category_code_1_0.png', {purge: true});

        const hash = crypto.createHash('md5').update(file);
        expect(hash.digest('hex')).to.matchSnapshot(this);
    });
});
