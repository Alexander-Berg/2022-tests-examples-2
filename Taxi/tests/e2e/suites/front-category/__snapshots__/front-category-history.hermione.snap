exports[`История изменений фронт-категории Скачивание изображения из истории фронт-категории 1`] = "ee878398a611e1df3ca98fa98d073f36";